docker/Dockerfile-php56

```diff
@@ -1,23 +1,55 @@
-FROM jolicode/php56:latest
+FROM php:5.6-alpine
 
 ENV SRC_DIR /usr/src/gaufrette
-WORKDIR ${SRC_DIR}/
 
-RUN sudo apt-get update && \
-    sudo apt-get install -y libssh2-1-dev && \
-    sudo rm -rf /var/lib/apt/lists/* && \
-    pecl install mongodb ssh2
+RUN apk add --no-cache --virtual .persistent-deps \
+        git \
+        zlib
 
-COPY composer.json ${SRC_DIR}/
-RUN sudo chown -R travis ${SRC_DIR} && \
-    composer global require hirak/prestissimo && \
-    composer update --prefer-lowest --prefer-stable && \
-    composer require microsoft/windowsazure:~0.4.3 && \
-    rm -rf ~/.composer/cache/*
+# PHP extensions
+ENV APCU_VERSION="4.0.11" \
```

Comment:
Do you plan to clean/fix the tests for the Apcu adapter? So far we didn't want to dedicate time to maintain deprecated adapters. It's not the best but we don't merge BC changes, so it's not really a problem.

Reply:
My bad, we don't even have an Apcu adapter. There's the Apc one, but it's deprecated so we deprecated the adapter too. It'll be removed in the next major.